src/components/CompanyCombo.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <span class="combo">
+    <span>
```

Comment:
Maybe give an icon for companies, just to maintain a consistency with the other clients in the list.

Reply:
Also make company names share the same color as client names.